content/docs/command-reference/remote/modify.md

```diff
@@ -185,11 +185,15 @@ parameters to customize the authentication method:
   $ dvc remote modify myremote use_ssl false
   ```
 
-- `ssl_verify` - whether or not to verify SSL certificates. By default SSL
-  certificates are verified.
+- `ssl_verify` - whether or not to verify SSL certificates, or a path to a
+  custom CA certificates bundle to do so (implies `true`). The certs in
+  [AWS CLI config](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html#cli-configure-files-settings)
+  (if any) are used by default.
```

Comment:
@rgvanwesep can you confirm the default is still the same after https://github.com/iterative/dvc/pull/6018#discussion_r635678930 ?

Reply:
Nvmd I guess the fall back is still `None` even if not explicit.